layers/+spacemacs/spacemacs-visual/packages.el

```diff
@@ -55,6 +55,15 @@
     :defer t
     :init (spacemacs/add-to-hooks 'hl-todo-mode '(text-mode-hook
                                                   prog-mode-hook))))
+    ;; (when (hl-todo-mode-enable-org-support)
+    ;;   (spacemacs/add-to-hooks 'org-mode-hook '(lambda () (hl-todo-mode -1)))
+    ;;   )))
+
+(defun spacemacs-visual/post-init-hl-todo ()
+  (message (configuration-layer/layer-usedp "org"))
+  (message (hl-todo-mode-disable-in-org-mode))
```

Comment:
The variable doesn't need parens to be evaulated.  Parens are for calling a function.  Also, I believe `(message ...)` takes a string as an argument.

Reply:
I'm a bit rusty in my lisp, but getting back into it.  I think if you want to use this as debug you probably want something like `(princ ...)`  I'm sure there's a better way.